docs/rules/jsx-first-prop-new-line.md

```diff
@@ -69,6 +75,27 @@ The following patterns are not considered warnings when configured `"multiline"`
 />
 ```
 
+The following patterns are considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
+    }}
+    bar />
+```
+
+The following patterns are not considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
```

Comment:
That would be handled by the existing `multiline` option. This is the `multiline-multiprop` option which does not warn because there is only 1 prop. 

I wanted this option to allow something like this:
```jsx
<Hello style={{
  'background': 'black',
}} />
```

but not
```jsx
<Hello style={{
  'background': 'black',
}} secondProp />
```